### PR TITLE
Bumping up to 1.10

### PR DESCRIPTION
**Pull request by wangyems**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: docs/Versioning.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
+| 1.10.0 | **1.10** down to 1.2 | 14 | 2 | 7 | Windows AI 1.10+ |
```
Comment (wangyems) [inline docs/Versioning.md:29]: Not sure if this line is correct

File: docs/python/README.rst [hunk 2]
```diff
@@ -8,6 +8,11 @@ For more information on ONNX Runtime, please see `aka.ms/onnxruntime <https://ak
 Changes
 -------
 
+1.10.0
+^^^^^
+
+Release Notes : https://github.com/Microsoft/onnxruntime/releases/tag/v1.10.0
```
Comment (tianleiwu) [inline docs/python/README.rst:14]: Shall we add release notes right before 1.10 release?

File: docs/Versioning.md (hunk 1 above)
Comment (tianleiwu) [reply docs/Versioning.md:29]: It's fine. It might be updated later if newer version of onnx comes out.

**Review by tianleiwu — DISMISSED** [review]

File: docs/Versioning.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
...
 | 1.9.0 | **1.9** down to 1.2 | 14 | 2 | 7 | Windows AI 1.9+ |
```
Comment (tianleiwu) [inline docs/Versioning.md:30]: Does 1.9.0 also support onnx 1.10 ?

Comment (wangyems) [reply docs/Versioning.md:30]: Yes it supports 1.10 from the release page

File: docs/python/README.rst (hunk 2 above)
Comment (wangyems) [reply docs/python/README.rst]: removed

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**